app/models/project_media.rb

```diff
@@ -58,8 +63,8 @@ def add_elasticsearch_data
     ms.team_id = p.team.id
     ms.project_id = p.id
     ms.set_es_annotated(self)
-    ms.status = m.last_status(self.project)
-    data = m.data(self.project)
+    ms.status = self.last_status
+    data = self.data
```

Comment:
Like `information`, `data` is too generic a name. We need a more descriptive name for what this thing is.